### PR TITLE
Feature/initial cd

### PR DESCRIPTION
**Pull request by bowlofstew**
This pull request (PR) adds support for release based branches for CD.  This will make it easier for other developers not familiar with the project to get up and running, reduce overall maintenance costs, etc per OKR #2.

This change is [Reviewable](https://reviewable.io/reviews/mozilla/subhub/476)

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    echo "No DEPLOY_ENV to set."
+    ;;
+esac
+
+if [ -z "$DEPLOY_ENV" ]; then
+      echo "Not deployinng"
+else
+      echo "Deploying to $DEPLOY_ENV"
+      doit deploy
+fi
```
Comment (The-smooth-operator) [inline bin/deploy.sh:28]: Missing new line

File: .travis.yml [hunk 2]
```diff
@@ -2,33 +2,38 @@ language: python
 dist: xenial
 sudo: required
 python:
- - "3.7"
+- '3.7'
```
Comment (The-smooth-operator) [inline .travis.yml:5]: Just out of curiosity, I see here you change `"` per `'`. But some of the new introduced parameters above are using `"`. Is this intended?

File: bin/deploy.sh (hunk 1 above)
Comment (bowlofstew) [reply bin/deploy.sh:28]: On it, thanks!

File: .travis.yml (hunk 2 above)
Comment (bowlofstew) [reply .travis.yml:5]: Good catch.  I will go over and make it overall consistent now.

**Pull request merged**